staging/kos/etcd-tls/cnf/etcd-client.cnf

```diff
@@ -0,0 +1,11 @@
+[req]
+req_extensions = v3_req
```

Comment:
I doubt that we need to specify `v3` in both the command line and the config file.

Reply:
Actually, since it refers to a non-empty section, I think we need this here, at least.